_posts/2019/2019-07-31-consistency.md

```diff
@@ -0,0 +1,108 @@
+If all buttons are e.g. structured as in our example, you could later on search for keys that contain `Button.label` and easily get an overview of all already existing UI texts for button labels.
+See?
+Consistent translation keys definitely support consistent UI texts.
+
+But as many different developers work on a software and create such translation keys, they will have their own style to create and name them.
+That's why you should invest time to [come up with a style guide for translation keys](https://developer.epages.com/blog/language-and-localization/why-you-should-invest-time-on-translation-keys/){:target="_blank"}.
+It leads to a company-wide consistent usage, and thus enables users to understand and categorize the keys more easily.
+Furthermore, developers have a given template and don’t need to spend extra time thinking about the correct term or structure.
+
+## Translation Memories
+
+Most of the time there is no one and only solution for translating a sentence into another language.
+There are several solutions.
+But a consistent translation for the same term or phrase will improve usability.
+
+As already stated in the introduction, it is kind of impossible to have all existing translations in mind.
+Especially if you work on larger or several different projects.
+That's exactly where translation memories jump in.
+They remind you of existing translations even if the source text is not exactly the same.
+By taking the suggestions from the translation memory into account, you can thus easily translate recurring terms and phrases consistently.
+
+## Glossaries
+
+I think we all agree that translation memories are a great way to support consistency in translations.
+But what about consistency in copywriting?
+If there is no text that the translation memory can compare, how should it be able to make suggestions?
+
+So, copywriters have a green field and can come up with whatever they want?
+That's not true either.
+Of course, they also need to take care of using the common terms and phrases.
+But it's a bit harder to find them.
+Glossaries can help here.
+They cannot only be used to explain certain terms and concepts to externals like translation agencies.
+By [maintaining a glossary](https://developer.epages.com/blog/language-and-localization/five-compelling-reasons-why-you-should-care-about-terminology/){:target="_blank"} you also facilitate the life of your copywriters.
+If they are e.g. wondering if they should say *user name* or *username*, or *ecommerce* or *e-commerce*, they can check the spelling in the glossary.
+This way the most important terms of your product will be spelled consistently throughout your software.
+
+## UI text guidelines
+
+Glossaries can support copywriters with the right spelling of a term.
+But what about whole sentences?
+Or the general grammar, and tone of voice?
+You'll need a guideline for these topics.
+The content of the guideline is totally up to you and your product.
```

Comment:
Would neutralize this instead of saying "you".
Also the whole below section should be neutral.